README.md

```diff
@@ -264,6 +264,20 @@ $ oneview-sdk-ruby create_from_file /my-server-profile.json
 $ oneview-sdk-ruby delete_from_file /my-server-profile.json
 ```
 
+##### Update resources by name:
+
+```bash
+$ oneview-sdk-ruby update FCNetwork FC1 -h linkStabilityTime:20  # Using hash format
+$ oneview-sdk-ruby update Volume VOL_01 -j '{"shareable": true}' # Using json format
+```
+
+##### Make REST calls:
+
+```bash
+$ oneview-sdk-ruby rest get rest/fc-networks
+$ oneview-sdk-ruby rest PUT rest/enclosures/<id>/configuration
```

Comment:
Ah. I would have made them the same, but I missed that (copy and paste). But yeah, case really doesn't matter, so users can use whatever they prefer